ui/webui/src/components/storage/MountPointMapping.jsx

```diff
@@ -207,11 +207,10 @@ const FormatColumn = ({ deviceData, handleRequestChange, idPrefix, request }) =>
 
     return (
         <Flex>
-            <Checkbox
-              id={idPrefix + "-checkbox"}
+            <Switch
+              id={idPrefix + "-switch"}
               isChecked={request.reformat}
               isDisabled={isRootMountPoint}
-              label={_("Reformat")}
```

Comment:
For making this accessible, you need to add an 'aria-labelled-by' and point to the Table header.
For making the Table header for the 'Format' have an id, I think you can add an 'id' property to the props in the line:
`{ title: _("Reformat"), props: { width: 20 } }, `

Reply:
In the end I added the switch directly to aria-label, because the tests didn't like the stitches without label or aria-label, aria-labelled wouldn't work